modules/react/icon/lib/Graphic.tsx

```diff
@@ -79,23 +109,56 @@ export const graphicStencil = createStencil({
   },
 });
 
+export const graphicImageStencil = createStencil({
+  vars: {
+    width: '',
+    height: '',
+  },
+  base: ({width, height}) => ({
+    '& [data-part="graphic-img"]': {
+      width,
+      height,
+      maxWidth: '100%',
+      maxHeight: '100%',
+    },
+  }),
+  modifiers: {
+    grow: {
+      true: {
+        '& [data-part="graphic-img"]': {
+          width: '100%',
+        },
+      },
+    },
+  },
+});
+
 export const Graphic = createComponent('span')({
   displayName: 'Graphic',
-  Component: ({grow, width, height, ...elemProps}: GraphicProps, ref, Element) => {
+  Component: (
+    {grow = false, width, height, src, srcset, alt, sizes, ...elemProps}: GraphicProps,
```

Comment:
`srcset` is not used anywhere. Is it supposed to be forwarded to the `img` element?

Reply:
updated.